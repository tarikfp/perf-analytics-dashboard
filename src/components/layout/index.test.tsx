import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Layout from '.';

test('layout renders correctly', () => {
  const tree = renderer.create(<Layout />);
  expect(tree).toMatchSnapshot();
});
