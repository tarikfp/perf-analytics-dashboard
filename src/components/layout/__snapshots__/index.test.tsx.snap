// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout renders correctly 1`] = `
<div
  className="MuiBox-root css-xff8c1"
/>
`;
